src/base.mli

```diff
@@ -63,7 +63,7 @@ val config :
   ?persistence:string ->
   ?persistence_name:string ->
   ?cookie_name:string ->
-  ?loaded:(Ojs.t -> unit) ->
+  ?loaded:(unit -> unit) ->
```

Comment:
It is a breaking change, so we should probably increase the major version number (i.e. move from 2.0 to 3.0).

Reply:
I will have to change the number version of the "Group" module PR too but I'll do it (and the tags will be renamed too).